### ISSUE TITLE
Make installation into a project more future proof

### ISSUE THREAD

**Issue by tovrstra** [theochem/cardboardlint#12]
It would be good that the instructions in the README keep working for a long time, even if we decide to structure the project differently, e.g. move code from the main script to a package with modules to keep it overseable.

We could do this with a `tools/bootstrap_run.sh` script (to be written) to be used as follows:
```bash
curl https://github.com/theochem/cardboardlint/tree/master/tools/bootstrap_run.sh | bash
```

Inside this script we can still decide how to organize things:
- short term: just download the main script and run it
- mid term: get the latest master, install and run
- long term: install with pip and run

We could also add a ```tools/bootstrap_install.sh``` to just install without running.

**Comment by tovrstra** [comment]
Also see theochem/horton#69.

**Comment by tovrstra** [comment]
Fixed in #22
